courses/util.py

```diff
@@ -133,6 +134,15 @@ def add_college_requirements(course, college_reqs):
         req.courses.add(course)
 
 
+def relocate_reqs_from_rests(rests, reqs):
```

Comment:
Might want to consider spelling out `restrictions` at the end of this function name, since `rest` is a little unclear